frontend/src/pages/Login.vue

```diff
@@ -112,6 +113,7 @@ const onSubmit = handleSubmit(async (values) => {
     })
     const { login: loginResponse } = result.data
     await store.dispatch('login', loginResponse)
+    store.commit('gmsActive', CONFIG.GMS_ACTIVE)
```

Comment:
Why store it in local store, when it already is in the config?
So when we disable GMS on server side and the user was still logged in, his frontend will try to show gms things, but the backend will deliver errors for all gms related requests?